source/_posts/2018-03-14-GoogleCast.md

```diff
@@ -0,0 +1,401 @@
+---
+title:  GoogleCast
+date: 2018-03-14 10:00:00
+tags: ios,google,googlecast
+authorIds:
+- anho
+categories:
+- iOS
+---
+
+Today, on the AppStore, you are able to find a rapidly increasing number of apps that will allow their users to watch video content. There might come a time when you will have to develop such an app, and functionality like GoogleCast and AirPlay is not only nice to have, but also expected by the end user.
+
+In this post we will tackle together the steps required in order to allow your users cast videos to a GoogleCast device.
+
+## Let's get started
+In order to be able to solely focus on integrating the GoogleCast SDK to our app, I have created a demo project that you can use in order to follow along.
+
+The project containt:
+
+- Added GoogleCast SDK via CocoaPods [Setup Instructions](https://developers.google.com/cast/docs/ios_sender_setup#xcode_setup)
+- An AVPlayer that allows users to play video content locally
+
+As well you will need to register your application to Google Cast SDK Developer Console at: https://cast.google.com/publish/#/signup
+
+You can download the demo project here: [Started Project](https://github.com/nodes-projects/cast-nodes-ios/tree/cast-demo-start)
```

Comment:
Have you made the repo public? Or maybe move it to nodes-ios instead

Reply:
I have made it public

Reply:
OK, you better check with @steffendsommer or someone else maybe....but personally I think the project should be in nodes-ios along all the other iOS projects